### PR TITLE
feat: return no Content when a user existed

### LINKED ISSUE #286: Fix: Cognito auth signup doesn't work
## Bug Description

Fix: Cognito auth signup doesn't work

## Steps to Reproduce

1. Go to 'Hosted signup UI'
2. Click on 'Google login'
3. see error in response `error_description: PreSignUp failed with error 2024-02-04T01:23:26.945Z fddd4e41-54b3-488c-b329-adeae0ecf465 Task timed out after 3.07 seconds. 
` but user data created in Dynamo not in Cognito user pool
4. do 1, and 2. again
5. see error in response `error_description: PreSignUp failed with error HTTP request failed with status 400: Bad Request. 
` nothing happened to cognito user pool
6. delete created data in Dynamo, and do 1. and 2. again
7. login suceeded

## Expected Behavior

after doing 1. and 2., login should be success.

## Additional Context

If I didn't do the 4. process, login couldn't be a success.

### PR DESCRIPTION
**Pull request by taniguchiiqqq**
## Overviews of implementation
changes to fix the signUp with cognito failure,
if I set duration more than equal 5s, cognito trigger call the hook lambda function twice. it cause somehow the pre-signup process succeed. so I changed the response when a user already exist.
In the case I call it only once, it cause time out error even though all process is done in 5 sec.

> Except for Custom sender Lambda triggers, Amazon Cognito invokes Lambda functions synchronously. When Amazon Cognito calls your Lambda function, it must respond within 5 seconds. If it doesn't and if the call can be retried, Amazon Cognito retries the call. After three unsuccessful attempts, the function times out. You can't change this five-second timeout value. For more information, see Lambda programming model in the AWS Lambda Developer Guide.
https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools-working-with-aws-lambda-triggers.html

## Review points
I assume it's not good solution at all, but for now I prioritized that the signing up works. Please let me know your opinion if it's acceptable or not.

### CODE REVIEW

**Review by hitohata — COMMENTED** [review]
Why did you change the return status? I think it will work if the response is 400.

**Comment by taniguchiiqqq** [comment]
OMG, That's true. I shouldn't have changed CreateANewUserService.ts, I will fix it. Thank you so much

File: backend/initializeUser/src/handler.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const handler = async (
 	const initializeUserService = new InitializeUserService();
 	const response = await initializeUserService.execute(initializeUserProps);
 
-	if (!(response.status === 201 || response.status === 204)) {
+	if (!(response.status === 201 || response.status === 400)) {
```
Comment (hitohata) [inline backend/initializeUser/src/handler.ts:27]: I think we only need to check whether the status code is 201 or not.

**Review by hitohata — APPROVED** [review]
LGTM

File: backend/initializeUser/src/handler.ts (hunk 1 above)
Comment (taniguchiiqqq) [reply backend/initializeUser/src/handler.ts:27]: I thought the same thing. But in that case, the lambda returns an error to Cognito when it is called twice, and then the User could not complete the signup.

**Pull request merged**